cmd/nerdctl/image_convert.go

```diff
@@ -245,10 +249,76 @@ func imageConvertAction(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	if finalize != nil {
+		ctx, done, err := client.WithLease(ctx)
+		if err != nil {
+			return err
+		}
+		defer done(ctx)
+		newI, err := finalize(ctx, client.ContentStore(), targetRef, &newImg.Target)
+		if err != nil {
+			return err
+		}
+		is := client.ImageService()
+		_ = is.Delete(ctx, newI.Name)
+		finimg, err := is.Create(ctx, *newI)
+		if err != nil {
+			return err
+		}
+		fmt.Fprintln(cmd.OutOrStdout(), "extra image:", finimg.Name)
```

Comment:
Is this string machine parsable?

Probably `--format=json` should be also available.
Can be another PR though.

Reply:
Added flag `--format=json` to print minimal information about the created images.

```console
# nerdctl image convert --format=json --oci --estargz --estargz-external-toc ghcr.io/stargz-containers/ubuntu:22.04 registry2:5000/ubuntu:22.04-ex
{
    "Image": {
        "Name": "registry2:5000/ubuntu:22.04-ex",
        "Digest": "sha256:3059dd5d9c404344e0b7c43d9782de8cae908531897262b7772103a0b585bbee"
    },
    "ExtraImages": [
        {
            "Name": "registry2:5000/ubuntu:22.04-ex-esgztoc",
            "Digest": "sha256:18c042b6eb8b208a8b97c88ad6d5186893141cb4ff43fab8c9bbb936eba8ae89"
        }
    ]
}
```

```console
# nerdctl image convert --format=json --oci --estargz ghcr.io/stargz-containers/ubuntu:22.04 registry2:5000/ubuntu:22.04-ex
{
    "Image": {
        "Name": "registry2:5000/ubuntu:22.04-ex",
        "Digest": "sha256:6d17b10f2d09267705f9500f07599cc2db57bee2c3175a1d0f91c0d88f43c9ec"
    },
    "ExtraImages": null
}
```